html-skeleton/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>HTML Skeletons</title>
+</head> 
+
+<body>Hello, HTML!</body>
+
```

Comment:
You should never put text directly in the body tags, try wrapping it in a `<p>` tag!
![image (30)](https://user-images.githubusercontent.com/52118958/100182497-a1864900-2e91-11eb-853c-0fc9ac32a6eb.png)